pinot-query-runtime/src/main/java/org/apache/pinot/query/service/server/QueryServer.java

```diff
@@ -159,37 +164,49 @@ public void submit(Worker.QueryRequest request, StreamObserver<Worker.QueryRespo
       return;
     }
 
-    try (QueryThreadContext.CloseableContext queryTlClosable = QueryThreadContext.openFromRequestMetadata(reqMetadata);
+    try (QueryThreadContext.CloseableContext qTlClosable = QueryThreadContext.openFromRequestMetadata(reqMetadata);
```

Comment:
(minor) The naming is quite confusing. What does `tl` stands for?

Reply:
Probable ThreadLocal, but I've just renamed them